packages/compendium-cli/.compendium-config.yml

```diff
@@ -0,0 +1,27 @@
+nodeTypes:
```

Comment:
These types should be rather general - not the domain-specific types

Reply:
trimmed it to just some general examples